src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -94,16 +95,17 @@ class ReportActionItemMessageEdit extends React.Component {
     /**
      * Update the value of the draft in Onyx
      *
-     * @param {String} newDraft
+     * @param {String} draft
      */
-    updateDraft(newDraft) {
-        this.textInput.setNativeProps({text: newDraft});
-        this.setState({draft: newDraft});
+    updateDraft(draft) {
+        const textWithEmojis = EmojiUtils.replaceEmojis(draft);
+        this.textInput.setNativeProps({text: textWithEmojis});
+        this.setState({draft: textWithEmojis});
 
         // This component is rendered only when draft is set to a non-empty string. In order to prevent component
         // unmount when user deletes content of textarea, we set previous message instead of empty string.
-        if (newDraft.trim().length > 0) {
-            this.debouncedSaveDraft(newDraft);
+        if (textWithEmojis.trim().length > 0) {
+            this.debouncedSaveDraft(textWithEmojis);
```

Comment:
Revert this after refactoring

Reply:
oh, I got your point, you are looking for minimal changes